hyp3_gamma/metadata/templates/insar/readme.md.txt.j2

```diff
@@ -102,6 +102,8 @@ The unwrapped interferogram uses a reference point to convert the wrapped 2-pi s
 
 The GeoTIFF contains the unwrapped phase difference values for each pixel, corresponding to the change in distance along the line of sight of the sensor. Positive values indicate movement away from the sensor (caused by subsidence or lateral movement away from the sensor) and negative values indicate movement towards the sensor (uplift or lateral movement towards the sensor). Note that deformation often includes both vertical and horizontal components, and additional processing or external reference data is required to calculate the relative contributions of vertical and lateral movement.
 
+The PNG file displays the unwrapped phase with a cyclic color ramp applied. Each color cycle (fringe) represents a phase difference of 6 pi, which equates to 1.5 times the sensor wavelength in line-of-sight displacement. Because each 2-pi difference is equivalent to half the wavelength of the sensor, each 6-pi fringe represents about 8.3 cm of displacement for these Sentinel-1 products.
```

Comment:
each 6-pi fringe represents about 8.3 cm of line of sight displacement for these Sentinel-1 products.